omegatrade-app/backend/app/models/company.model.js

```diff
@@ -0,0 +1,196 @@
+        if (stockResult[0]) {
+            var stockRows = stockResult[0].map((row) => row.toJSON());
+            response.stocks = stockRows;
+        }
+        cb(null, response)
+    } catch (error) {
+        cb(error, null)
+    }
+};
+
+Company.create = async function (params, result) {
+    try {
+        params.companyId = uuidv4();
+        params.created_at = 'spanner.commit_timestamp()';
+        await database.table('companies').insert(params);
+        result(null, params.companyId);
+    } catch (error) {
+        result(true, null);
+    }
+};
+
+Company.createData = async function (params, result) {
+    try {
+        let companyId = uuidv4();
+        await database.table('companies').insert({
+            "companyName": params.name,
+            "companyShortCode": params.symbol,
+            "companyId": companyId,
+            "created_at": 'spanner.commit_timestamp()'
+        })
+        result(null, companyId)
+    } catch (error) {
+        result(true, null)
+    }
+};
+
+Company.createStockData = async function (stockData, result) {
+    try {
+        await database.table('companyStocks').insert(stockData)
+        result(null, true);
+    } catch (error) {
+        console.log('stockDataErr', error)
+        result(error, false);
+    }
+};
+
+Company.getCompany = async function (companyId, cb) {
+    console.log(companyId)
+    try {
+        const query = {
+            sql: 'select * from companies where companyId = @companyId',
+            params: {
+                companyId: companyId
+            }
+        };
+        let result = await database.run(query);
+        if (result[0]) {
+            var rows = result[0].map((row) => row.toJSON());
+            cb(null, rows)
+        } else {
+            cb(null, null);
+        }
+    } catch (error) {
+        console.log('issue', error)
+        cb(error, null)
+    }
+};
+
+Company.checkCompany = async function (param) {
+    companyName = param.companyName;
+    companyShortCode = param.companyShortCode;
+    try {
+        const query = {
+            sql: 'select * from companies where companyName = @companyName or companyShortCode = @companyShortCode',
+            params: {
+                companyName: companyName,
+                companyShortCode:companyShortCode
+            }
+        };
+        let result = await database.run(query);
+        console.log(result);
+        if (result[0]) {
+            var data = result[0].map((row) => row.toJSON());
+            return {status:true,data:data}
+        } 
+    } catch (error) {
+        return {status:false}
+    }
+};
+
+Company.delete = async function (companyId, cb) {
+    database.runTransaction(async (err, transaction) => {
+        if (err) {
+            cb(err, null)
+            return;
+        }
+        try {
+            const [rowCount] = await transaction.runUpdate({
+                sql: "DELETE FROM companies WHERE companyId = @companyId",
```

Comment:
Is there a specific reason that you are using DML here instead of mutations (that is: `database.tables('companies').deleteRows`)?

Mutations generally speaking perform better than DML statements, and should therefore normally be preferred above DML statements, unless DML statements are necessary for other reasons (such as being able to read your own writes within the same transaction).